tmt/base.py

```diff
@@ -3110,12 +3111,12 @@ def init(
                     path = tree.root
                 except fmf.utils.GeneralError as error:
                     raise tmt.utils.GeneralError(
-                        f"Failed to initialize tree in '{path}': {error}")
-                echo(f"Tree '{tree.root}' initialized.")
+                        f"Failed to create '{fmf_dir}'") from error
+                echo(f"Initialized metadata dir '{fmf_dir}'")
```

Comment:
I'm not sure if `metadata directory` fits here well. The `.fmf` directory actually does not contain any metadata. Unlike `git`, `fmf` just marks root of the metadata tree and stores a `version` file there.

Reply:
I agree that `Initialized fmf tree root '{fmf_dir}'` is the most clear one.

Reply:
Ok, let's use consistently `fmf tree root` across all messages: Done in 7997084b.